package.json

```diff
@@ -1,43 +1,47 @@
 {
```

Comment:
can you explain what's going on with this file?

Reply:
(I know the preview is a little borked, but my question does refer to the whole file 😄 )

Reply:
I didn't touch it. I think the reason why it is changed is because I just edited the current version of pros-atom that I had installed, then committed and uploaded that. I might have had a slightly outdated version that had a different package.json. You can just ignore any changes made to this file, if that is possible.

Reply:
I probably should have only 'git add'ed the files I changed. Like I said, this is my first pull request, so I might not have done everything right :stuck_out_tongue: